### PR TITLE
Fix #43, version bump, update change log, and make changelog markdown

### LINKED ISSUE #43: Password fields should be `:validate => :password`
https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/blob/master/lib/logstash/plugin_mixins/rabbitmq_connection.rb#L65

### PR DESCRIPTION
**Pull request by jakelandis**
Fixes #43

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+## 5.0.1
+   - Relax constraint on march hare to allow 3.x [#44](https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/issues/44)
+   - Validate validate :ssl_certificate_password as a password [#43](https://github.com/logstash-plugins/logstash-mixin-rabbitmq_connection/issues/43)
```
Comment (jakelandis) [inline CHANGELOG.md:3]: Everything below is the same content... just formatted as markdown.

**Review by andrewvc — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (yaauie) [reply CHANGELOG.md:3]: intentional doubling? `Validate validate`

Comment (jakelandis) [reply CHANGELOG.md:3]: doh! ... i'll sneak the fix directly into master since this has already been merged.

**Pull request closed without merging**